### ISSUE TITLE
Feature Request #7

### ISSUE THREAD

**Issue by S3RAPH-1M** [devinSpitz/PavlovRconWebserver#85]
Allow the groups permissions for each command to be changed.

**Comment by devinSpitz** [comment]
yeah thats possible and it can be combined with the buckets maybe. So you can make buckets of commans and make it possible for other roles to use it.
